src/GravityFormsExtensions.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+namespace P4\MasterTheme;
+
+/**
+ * Class P4\MasterTheme\GravityFormsExtensions
+ * The Gravity form plugin extension, use to add custom functionality like Planet4 form type.
+ */
+class GravityFormsExtensions {
+	/**
+	 * @var string The default gravity form type.
+	 */
+	public const DEFAULT_GF_TYPE = 'other';
+
+	/**
+	 * @var array The Planet4 Gravity form types.
+	 */
+	public const P4_GF_TYPES = [
+		[
+			'label' => 'Other',
+			'value' => 'other',
+		],
+		[
+			'label' => 'Petition',
+			'value' => 'petition',
+		],
+		[
+			'label' => 'Email Signup',
+			'value' => 'email-signup',
+		],
+		[
+			'label' => 'Quiz/Poll',
+			'value' => 'quiz/poll',
```

Comment:
I think that changing to `quiz-poll` will be more consistent with the rest of values. What do you think?